utils/python/CIME/SystemTests/system_tests_common.py

```diff
@@ -199,6 +199,10 @@ def run_indv(self, suffix="base", st_archive=False):
             infostr += ", with restarts every %d %s"%(rest_n, rest_option)
         logger.info(infostr)
 
+
```

Comment:
@jgfouca is right that much of my desire was satisfied with his signal handling changes. I'd still support differentiating between PEND and RUN if it's feasible.